helm/grafana/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+description: Grafana instance for kube-prometheus
+engine: gotpl
+maintainers:
+- email: weiwei.inf@gmail.com
```

Comment:
Yes, we are a huge fan of prometheus-operator & kube-prometheus, we use kube-prometheus(with some modify, mainly alert rules) to monitor our k8s cluster. And we need this grafana app for our k8s system admin to see k8s monitor dashboard, and we are also planing to deploy this grafana app for each product line in their own namespace with some pre installed dashboard(like their pod metrics), so our user can see their own pod's metrics and they can also config their own dashboard for their business metrics.

Thanks for providing prometheus-operator & kube-prometheus to the community, saved a lot of our times :)

I and my colleague are very glad to make contribution to this project, if need help feel free to @weiwei04 @cofyc @WIZARD-CXY :)